### PR TITLE
add readme for the frontend

### PR DESCRIPTION
**Pull request by gcmalloc**
README didn't had any section for the frontend, add those.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,14 @@ The import order and location be fixed to match the pep8 using ![isort](https://
 
 # Local Dev environment
 
-First, clone this repository, then git@github.com:enermaps/Hotmaps-toolbox-service.git in the same directory.
+First, clone this repository, 
+then git@github.com:enermaps/Hotmaps-toolbox-service.git in the same directory.
```
Comment (gcmalloc) [inline README.md:25]: change then for a capitlized version:

```
Then ...
```

Comment (cedricserugendo) [reply README.md:25]: The code is perfect like that

File: README.md [hunk 2]
```diff
@@ -39,7 +42,7 @@ this will start the following set of services:
 The local postgresql will be available at 127.0.0.1:5432 
 the geoserver at http://127.0.0.1:8000/geoserver
 the api server at http://127.0.0.1:9000/api
-the frontend at http://127.0.0.1:7000/api
+the frontend at http://127.0.0.1:7000/
 
 You can setup the geoserver if you started it for the first time by running:
```
Comment (cedricserugendo) [inline README.md]: are these lines useful?

Comment (gcmalloc) [reply README.md]: Yes, you can check the rest of the README by clicking the expand button below the line numbers.

Comment (cedricserugendo) [reply README.md]: ok, i got it

**Pull request merged**